repose-aggregator/build.gradle

```diff
@@ -3,8 +3,23 @@ import java.text.SimpleDateFormat
 
 apply plugin: 'org.asciidoctor.convert'
 
+dependencies {
+    asciidoctor 'org.asciidoctor:asciidoctorj-diagram'
+}
+
 asciidoctor {
-    attributes 'source-highlighter': 'coderay'
+    requires 'asciidoctor-diagram'
+    attributes 'source-highlighter': 'coderay',
+               'stylesdir': "${project.projectDir}/src/docs/asciidoc/_stylesheets",
+               'stylesheet': 'repose.css',
+               'imagesoutdir': "${project.buildDir}/asciidoc/html5/images",
```

Comment:
Indeed. When I initially added it, the generated images were already cached in the build directory, so I didn't catch it at first.

I'm kind of annoyed, actually. From what I can tell, while you can specify the output directory, there isn't a way to tell asciidoctor-diagram to use that as the input directory for the image. I wanted it to be smarter. 😞